mibs/lenovo/LENOVO-PRODUCTS-MIB

```diff
@@ -10,29 +7,29 @@ IMPORTS
         FROM LENOVO-SMI-MIB;
 
 lenovoProductsMIB MODULE-IDENTITY
-    LAST-UPDATED    "201705010000Z"
```

Comment:
Are you sure you want to replace current mib file with an older one ? 
Renaming it properly is a good think, reverting to an older file usually not.

Reply:
Ah, I did not check that, because for my test system (from 2019) the 2016er version was the most recent one and I assumed that the other file was older. Fixed!